scripts/helpers.ts

```diff
@@ -128,7 +128,8 @@ export async function queryContract(terra: LCDClient, contractAddress: string, q
 
 export async function deployContract(terra: LCDClient, wallet: Wallet, admin_address: string, filepath: string, initMsg: object) {
     const codeId = await uploadContract(terra, wallet, filepath);
-    return await instantiateContract(terra, wallet, admin_address, codeId, initMsg);
+    let resp = await instantiateContract(terra, wallet, admin_address, codeId, initMsg);
+    return {"contract_codeID": codeId, "contract_address": resp.shift()}
```

Comment:
Keep it as it was. There are special cases for getting inner instantiated contract addresses, for example: https://github.com/astroport-fi/astroport-core/blob/b02abbdc9c7a13d438ce4932016f7b74c19b27d7/scripts/deploy_core.ts#L84